uaclient/cli.py

```diff
@@ -911,14 +944,17 @@ def action_enable(args, *, cfg, **kwargs):
             )
         )
         tmpl = ua_status.MESSAGE_INVALID_SERVICE_OP_FAILURE_TMPL
+        event.services_failed(entitlements_not_found)
         raise exceptions.UserFacingError(
             tmpl.format(
                 operation="enable",
                 name=", ".join(entitlements_not_found),
                 service_msg=service_msg,
             )
         )
+        ret = False
```

Comment:
I'm not sure we'll ever reach this line because of the `raise` above

Reply:
Yes, this is just leftover code. Good catch